documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -3242,6 +3246,10 @@ See <<oracle-property-log-mining-transaction-retention-hours, `log.mining.transa
 |`long`
 |The number of committed transactions in the transaction buffer.
 
+|[[oracle-streaming-metrics-number-of-oversized-transactions]]<<oracle-streaming-metrics-number-of-oversized-transactions, `+NumberOfOversizedTransactions+`>>
+|`long`
+|The number of oversized transactions in the transaction buffer.
```

Comment:
```suggestion
|The number of transactions that were discarded because their size exceeded <<oracle-property-log-mining-buffer-memory-transaction-events-threshold, `log.mining.buffer.memory.transaction.events.threshold`>>.
```
Note that I use the existing property name as described in the documentation from your original commit but this should be aligned with the suggestion above that we unify this to be implementation agnostic.